core/build.gradle

```diff
@@ -2,17 +2,17 @@ apply plugin: 'com.android.library'
 apply plugin: 'bintray-release'
 
 dependencies {
-    compile 'com.android.support:support-v4:19.1.0'
+    compile 'com.android.support:support-v4:23.0.1'
 }
 
 android {
     defaultConfig {
         minSdkVersion 14
-        targetSdkVersion 19
+        targetSdkVersion 22
```

Comment:
I put the other one down to 22 - this PR is not permissions model related so 22 felt more "lalala not my [pr is singing] problem"